app/helpers/curation_concerns/file_set_helper.rb

```diff
@@ -7,9 +7,9 @@ def parent_path(parent)
     end
   end
 
-  def media_display(file_set, locals = {})
-    render media_display_partial(file_set),
-           locals.merge(file_set: file_set)
+  def media_display(presenter, locals = {})
```

Comment:
@jeremyf this media_display helper is being used on both the work and the file set display. for each we are passing a file_set presenter after this PR.  Before the PR it was getting a file set object or a file set presenter depending on where it was called from.  I changed the parameter to explicitly say we want a presenter.  I do not think it matters what type of presenter it is as long as it responds to the mime type methods (image? audio?...) and has a to_s method.